client/extensions/woocommerce/components/sparkline/style.scss

```diff
@@ -0,0 +1,12 @@
+// Sparkline
+// Making things sparky
+.sparkline {
```

Comment:
There should be a `width: 100%` here. This way when the placeholder box gets rendered, `state.width` can be called via `componentDidMount` and the svg width set appropriately.